node/storage-cleaner/src/util.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Calculate and return the expiry date based on the retention period.
+ * @returns {string} The calculated expiry date in ISO format
+ */
+export function getExpiryDate() {
+  const retentionPeriod = +(process.env.RETENTION_PERIOD_DAYS ?? 30);
+  const expiryDate = new Date();
+  expiryDate.setDate(expiryDate.getDate() - retentionPeriod);
+  return expiryDate.toISOString();
+}
+
+/**
+ * Throws an error if any of the keys are missing from the object
+ * @param {*} obj
+ * @param {string[]} keys
+ * @throws {Error}
+ */
+export function throwIfMissing(obj, keys) {
+  const missing = [];
+  for (let key of keys) {
+    // using obj[key] != 0 for case when the value of key will be 0
+    if (!(key in obj) || (!obj[key] && obj[key] != 0)) {
```

Comment:
I think this condition can be simplified:
`!(key in obj && obj[key] !== 0)`

Can we also remove the comment?

Reply:
Done